<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dock_Background</class>
 <widget class="QDockWidget" name="Dock_Background">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop-wallpaper"/>
  </property>
  <property name="windowTitle">
   <string>Background</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>227</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Color_Selector" name="color_selector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="color">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="check_image">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Image</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>File</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="text_file"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="button_browse">
          <property name="text">
           <string>Browse...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Scale</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spin_scale">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1600.000000000000000</double>
          </property>
          <property name="value">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="button_move">
          <property name="text">
           <string>Move...</string>
          </property>
          <property name="icon">
           <iconset theme="transform-move">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Color_Preview</class>
   <extends>QWidget</extends>
   <header>color_preview.hpp</header>
  </customwidget>
  <customwidget>
   <class>Color_Selector</class>
   <extends>Color_Preview</extends>
   <header>color_selector.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
